### PR TITLE
Microverse first project

### PR DESCRIPTION
**Pull request by hislordshipprof**
In this project ;
Created an Html template
Created Css style sheet
Committed each of the files created individually
Created a test,md file and added it to gitignore

### CODE REVIEW

**Review by phelian23 — CHANGES_REQUESTED** [review]
Hi @hislordshipprof, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ Correct use of Github Flow
✅ Good commit messages
✅ Descriptive PR

## Required Changes ♻️
- [ ] The linters file is not set up correctly. Kindly follow the instructions [here](https://github.com/microverseinc/linters-config) to set up the linters correctly.
- [ ] Please use [this template](https://github.com/microverseinc/readme-template) to make you README.md file descriptive, but remember to customize it to your project.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by aimemalaika — APPROVED** [review]
Hi @hislordshipprof, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,397 @@
...
+                  </li>
+                  <li>
+                    <button type="submit" value="Submit" class="btn">
+                      Get In Touch
+                    </button>
+                  </li>
+                </ul>
+              </form>
+            </div>,
+          </section>
```
Comment (Mre55) [inline index.html]: Kindly, stick to the design provided as much as possible. This section has a background image please add the image as background inside your CSS file. And also it has a border-radius at the left top of the section. Match the text font-weight as the design.

| Your Design | Figma Design |
|-------------|---------------|
| image | image |

File: index.html [hunk 2]
```diff
@@ -0,0 +1,397 @@
...
+                      name="comment"
+                      placeholder="Write your message here"
+                      maxlength="500"
+                      required
+                    ></textarea>
+                  </li>
+                  <li>
+                    <button type="submit" value="Submit" class="btn">
+                      Get In Touch
+                    </button>
```
Comment (Mre55) [inline index.html:387]: Please match the design for your `Get In Touch` button` it should be somewhere in the middle between the text area filed and the bottom end. Currently, it's more to the bottom kindly edit its position and the button text spacing and alignment.

| Your Design | Figma Design |
|-------------|---------------|
| image | image |

**Review by Mre55 — CHANGES_REQUESTED** [review]
**Hi @hislordshipprof,**

You have done a great job so far! 💪 👏 
You still need to work on a few issues before moving on to the next project, but you're almost there!

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 

## Required Changes ♻️🛠️ 
_Check the comments under the review._




## Optional suggestions 🟡 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rotimiazeez — APPROVED** [review]
# Hi @hislordshipprof :wave, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**